### ISSUE TITLE
Scroll header is not working 

### ISSUE THREAD

**Issue by smanojarun** [mukyasa/MMGooglePlayNewsStand#5]
Hi,

I have liked the UI and tried this project. Scroll header is not working in this project. Have to implement the code to make it work? But the scroll delegate is calling. Can you guys please advice?

**Comment by mukyasa** [comment]
check the ObjCPlayStand Demo its working perfectly.

**Comment by smanojarun** [comment]
Yes i have tried. In App store version , when scroll down and then scroll up the header view is displaying.. But in this source , when scroll up at the begining only the header is showing and later the header is not showing when scroll down and then scroll up.. Please advice. Thanks.
